src/modules/user/user.schema.ts

```diff
@@ -0,0 +1,49 @@
+import { ObjectId } from 'mongodb'
+import { UserRole, UserVerifyStatus } from './user.enum'
+
+interface UserType {
```

Comment:
when i create account, i can pick number phone, or email, so one of them is optional